### ISSUE TITLE
[BUG] rtt in sample data returned from listener callback is in seconds when client is uing Firefox

### ISSUE THREAD

**Issue by adamcarchibald** [twilio/twilio-client.js#102]
- [x] I have verified that the issue occurs with the latest twilio.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-client.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-client.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/TwilioDevEd/client-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

```js
connecton.on('sample', (sample) => {
  console.log('RTT:', sample.rtt);
});
```

**Expected behavior:**

Rtt is reported in milliseconds

**Actual behavior:**

<img width="204" alt="Screen Shot 2019-11-20 at 11 43 39 AM" src="https://user-images.githubusercontent.com/16963265/69258761-2063ea00-0b8b-11ea-84ef-033c65cc915a.png">

**Software versions:**

 - [ ] Browser(s): Firefox Developer Edition 71.0b11
 - [ ] Operating System: OSX 10.14.6 
 - [ ] twilio.js:  flex-ui: 1.14.1,  twilio: 3.37.1, twilio-client: 1.9.5
 - [ ] Third-party libraries (e.g., Angular, React, etc.): React = 16.11.0, Redux = 3.7.1, Typescript = 3.7.2

**Comment by aymenn** [comment]
Thank you @adamcarchibald 

According to https://www.w3.org/TR/webrtc-stats/#dom-rtcremoteinboundrtpstreamstats-roundtriptime it should be in seconds. Ideally, we should be consistent with the standard. I think should add a new field rttSeconds to not break any existing apps. What do you think?

**Comment by adamcarchibald** [comment]
That sounds excellent to me 👍

**Comment by charliesantos** [comment]
Jira [CLIENT-7014](https://issues.corp.twilio.com/browse/CLIENT-7014)

**Comment by charliesantos** [comment]
Fixed in https://github.com/twilio/twilio-client.js/pull/125 and will be included in 1.11. Please watch out for the release.
